lib/desired-caps.js

```diff
@@ -10,7 +10,7 @@ let desiredCapConstraints = _.defaults({
     isString: true
   },
   wdaLocalPort: {
-    isNumber: true
+    isNumber: true,
```

Comment:
there are no trailing commas for other parameters. Do we follow any rule on this?